src/iarc7_motion/iarc_tasks/go_to_roomba_task.py

```diff
@@ -0,0 +1,190 @@
+import math
+import rospy
+import tf2_ros
+import tf2_geometry_msgs
+import threading
+
+from geometry_msgs.msg import TwistStamped
+from geometry_msgs.msg import Point
+from geometry_msgs.msg import PointStamped
+
+from task_utilities.acceleration_limiter import AccelerationLimiter
+
+from .abstract_task import AbstractTask
+from iarc_tasks.task_states import (TaskRunning,
+                                    TaskDone,
+                                    TaskCanceled,
+                                    TaskAborted,
+                                    TaskFailed)
+from iarc_tasks.task_commands import (VelocityCommand, NopCommand)
+
+from translate_stop_planner import TranslateStopPlanner
+
+class GoToRoombaState(object):
+    init = 0
+    translate = 1
+
+class GoToRoombaTask(AbstractTask):
+
+    def __init__(self, task_request):
+        super(GoToRoombaTask, self).__init__()
+
+        self._roomba_id = task_request.frame_id + '/base_link'
+
+        if self._roomba_id is None:
```

Comment:
This check will always pass; rather it needs to check to see if roomba ID is equal to ‘/base_link‘

Reply:
Changed the statement to check if it is equal to `'/base_link'`